cmd/res/profiles/ED_ID1.yaml

```
@@ -0,0 +1,114 @@
+name: "ED_ID1"
+manufacturer: "HCL Corp."
+model: "Some 3rd party app sending data"
+labels:
+ - "rest"
+ - "ED_ID1"
+description: "REST Device"
+
+deviceResources:
+  - name: jsonRes
```

Please remove `Res` from all the resource names
```suggestion
  - name: json
```